sources/reference/tutorials/batch.md

```diff
@@ -0,0 +1,608 @@
+# Running Chaos Toolkit experiments as AWS Batch Jobs
+
+It is common when using AWS for hosting your infrastructure that you'll have
+strict security policies in place. These policies will usually only allow for
+internal traffic within AWS, amongst various other things. A question we're
+asked a lot is `can I run Chaos Toolkit from AWS, to run against AWS?`.
+The answer is simply, yes, you can.
+
+## Why Batch?
+
+You may have followed our [Running Chaos Toolkit from an EC2 instance][] guide
+and wonder why we would write a guide for Batch - Batch has some benefits
+over EC2:
+
+[Running Chaos Toolkit from an EC2 instance]: ../containerising
```

Comment:
I believe the link should be ../ec2.md

Reply:
Good spot

Reply:
@CharlieMoon37 I've removed that specific link, there was one already defined later on that it'l use